master/buildbot/test/integration/test_custom_buildstep.py

```diff
@@ -12,9 +12,8 @@
 # Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 #
 # Copyright Buildbot Team Members
-from StringIO import StringIO
-
 import mock
+from future.utils import PY3
```

Comment:
Please add an empty line before this one.

My suggestion is to move `import mock` at the very end...